src/main/kotlin/com/cognifide/gradle/aem/base/vlt/CleanTask.kt

```diff
@@ -15,7 +15,13 @@ open class CleanTask : AemDefaultTask() {
     init {
         description = "Clean checked out JCR content."
 
-        beforeExecuted(config.syncTransferTaskName) { runner.cleanBeforeCheckout() }
+        project.run {
```

Comment:
project.gradle.taskGraph.whenReady {
    if (it.contains("aemClean")) {
                tasks.getByName(config.syncTransferTaskName).doFirst {
                    runner.cleanBeforeCheckout()
                }
    }
}